### PR TITLE
try explicitly handle PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by igalic**
put PULL_REQUEST_TEMPLATE in its own variable, this way our body logic
just works™
None ← except for this bit. Especially when that thing is just turned into "b'Hello, World!\\n'"

### CODE REVIEW

File: lib/gitspindle/github.py [hunk 1]
```diff
@@ -1170,29 +1170,30 @@ def pull_request(self, opts):
             print("Pull request %d created %s" % (pull.number, pull.html_url))
             return
 
-        body = self.find_template(repo, 'PULL_REQUEST_TEMPLATE')
+        pr_template = self.find_template(repo, 'PULL_REQUEST_TEMPLATE')
```
Comment (igalic) [inline lib/gitspindle/github.py]: `find_template()` already does `try_decode()`, however, it's not turning our bytes into an actual (Unicode) string.

File: lib/gitspindle/github.py [hunk 2]
```diff
@@ -1170,29 +1170,30 @@ def pull_request(self, opts):
...
+            body = ""
 
         body += """
 # Requesting a pull from %s/%s into %s/%s
 #
+%s
 # Please enter a message to accompany your pull request. Lines starting
 # with '#' will be ignored, and an empty message aborts the request.
-#""" % (repo.owner.login, src, parent.owner.login, dst)
+#""" % (repo.owner.login, src, parent.owner.login, dst, try_decode(pr_template))
```
Comment (igalic) [inline lib/gitspindle/github.py]: this try_decode() call here is superfluous, and is what causes the `None` output in this very pull request :)

**Comment by seveas** [comment]
Fixed slightly differently in 90cce34

**Comment by igalic** [comment]
thanks 💜 @seveas

**Pull request closed without merging**